static/js/webrtc.js

```diff
@@ -234,22 +240,38 @@ var rtc = (function() {
           videoContainer.addClass('local-user');
           video.muted = true;
         }
-        self.addInterface(userId);
+        self.addInterface(userId, stream);
       }
       if (stream) {
         attachMediaStream(video, stream);
       } else if (video) {
         $(video).parent().remove();
       }
     },
-    addInterface: function(userId) {
+    addInterface: function(userId, stream) {
       var isLocal = userId == self.getUserId();
       var videoId = "video_" + userId.replace(/\./g, "_");
       var $video = $("#" + videoId);
 
+      ///////
+      // Mute button
+      ///////
```

Comment:
This function has become very large. I thought about a cleanup processes but it would require a bit of time to test to make sure we don't break any variable capture stuff. Probably worth doing it later.

Reply:
Heh, now we have actual tests written. Should be easier. But probably for another PR.